packages/fxa-payments-server/src/index.scss

```diff
@@ -34,7 +34,129 @@ h3.billing-title {
   }
 }
 
-#main-content {
+// starting to replace the main-content id with this class
+.main-content {
+  display: flex;
+  padding-top: 0;
+
+  .input-row {
+    // Stripe element class names
+    // https://stripe.com/docs/stripe-js/reference#the-element-container
+
+    .StripeElement {
+      @include input-element();
+
+      html[dir='ltr'] & {
+        padding: 0 0 0 $input-left-right-padding;
+      }
+
+      html[dir='rtl'] & {
+        direction: ltr;
+        padding: 0 $input-left-right-padding 0 0;
+        text-align: right;
+      }
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+
+
+    .StripeElement--focus,
+    .StripeElement--focus:hover {
+      border-color: $input-border-color-focus;
+      box-shadow: 0 0 0 3px rgba($blue-50, 0.3);
+    }
+
+    .tooltip + .StripeElement--empty,
+    .StripeElement--invalid,
+    .StripeElement--invalid:hover {
+      border-color: $error-background-color;
+
+      &.StripeElement--focus {
+        box-shadow: 0 0 0 3px rgba($error-background-color, 0.3);
+      }
+    }
+
+    .label-text {
+      color: #686869;
+      display: block;
+      font-size: 13px;
+      font-weight: 500;
+      margin-bottom: 7px;
+      text-align: left;
+    }
+
+    .label-text.checkbox {
+      color: #0c0c0d;
+      font-size: 13px;
+      font-weight: normal;
+      letter-spacing: normal;
+      line-height: 21px;
+    }
+
+    // HACK: stop the checkbox from changing width on focus
+    input[type='checkbox']:focus,
+    input[type='checkbox']:not(:checked):focus,
+    input[type='checkbox']:checked:focus {
+      border: 1px solid transparent;
+    }
+  }
+
+  .input-row--checkbox {
+    label {
+      display: flex;
+      flex-direction: row;
+
+      input[type='checkbox'] {
+        flex: 0 0 18px;
+        position: relative;
+        top: 2px;
+
+        &:checked {
+          background-image: url('images/check.svg');
+        }
+      }
+
+      .label-text {
+        display: block;
+        margin-left: 2px;
+      }
+    }
+  }
+
+  .input-row-group {
+    align-items: flex-end;
+    display: flex;
+    flex-direction: row;
+
+    @include respond-to('simpleSmall') {
```

Comment:
Old mixin! Same as line 145 (unless this will be addressed in that cleanup issue we made a few days ago)

Reply:
I believe these breakpoint replacements are outside the scope of these updates. This change converting and id to a classname. I think we should only add the new breakpoint style and update the old breakpoints as tech debt issues. Since rewriting all of this as mobile first can turn into a huge effort

Reply:
Some of the old breakpoints are mobile-first but yeah, not all of them. 😓 I had messaged with @johngruen a few weeks ago about using those new breakpoints in the payments server for this project since there's a ton of CSS changes and he said it sounded like a good idea but if that feels like a high LoE / out of scope then we can do it in a follow-up. We'll have to comb through payments CSS again anyway when we set a standard for styling conventions (I keep waiting on seeing the settings redesign visual designs before proposing something). I think it's probably a good idea to make it consistent either way, if you've used `@include mi-width` anywhere we probably want to change it to `respond-to` and an older breakpoint.

Reply:
created follow up issue in jira